e2e/template_e2e_tests/run_e2e_test.sh

```diff
@@ -141,26 +141,43 @@ print_api_test_result() {
   fi
 }
 
+clean_up_gke() {
+  cd $BASE_DIR/terraform/stages/gke
+  terraform init -reconfigure -backend-config=bucket=$TF_BUCKET_NAME
```

Comment:
Why is `terraform init` needed here (as it should already to pointing to the remote state file)

Reply:
This is also intentional. Since we store the all the TF states in the bucket, this is to in case somehow you lost the state locally, and wish to restore the TF state in a different workspace (or laptop)